src/processor/handler.rs

```diff
@@ -211,15 +211,18 @@ pub trait TransactionContext {
     ///
     /// * `address` - address of where to store the data
     /// * `data` - payload is the data to store at the address
-    fn set_state_entry(&self, address: String, data: Vec<u8>) -> Result<(), ContextError>;
+    fn set_state_entry(&self, address: String, data: Vec<u8>) -> Result<Vec<String>, ContextError>;
 
     /// set_state_entries requests that each address in the provided map be
     /// set in validator state to its corresponding value.
     ///
     /// # Arguments
     ///
     /// * `entries` - entries are a hashmap where the key is an address and value is the data
-    fn set_state_entries(&self, entries: HashMap<String, Vec<u8>>) -> Result<(), ContextError>;
+    fn set_state_entries(
+        &self,
+        entries: HashMap<String, Vec<u8>>,
+    ) -> Result<Vec<String>, ContextError>;
```

Comment:
Is there any scenario where something isn't successfully set _without_ it being an error? If not, then the resulting list of addresses set will always be the the same list of addresses sent.

Reply:
Changed the return type back to Ok(())